### PR TITLE
Smartadserver Bid Adapter: Update default value of startDelay video parameter

### PR DESCRIPTION
**Pull request by krzysztofequativ**
## Type of change
[x] Feature

## Description of change
The **startDelay** parameter is set to **1** by default.

### CODE REVIEW

File: modules/smartadserverBidAdapter.js [hunk 1]
```diff
@@ -132,18 +132,19 @@ export const spec = {
...
+        case 0:
+          return 1;
+        case -1:
+          return 2;
+        case -2:
+          return 3;
       }
     }
-    return 2;// Default value for all exotic cases set to bid.params.video.startDelay midroll hence 2.
+    return 1; // SADR-5619
```
Comment (patmmccann) [inline modules/smartadserverBidAdapter.js:147]: this appears wrong; what do you do if videoMediaType is say, 180? In this case, you appear to ignore the declaration and guess that it is preroll?

**Review by patmmccann — CHANGES_REQUESTED** [review]

File: modules/smartadserverBidAdapter.js (hunk 1 above)
Comment (krzysztofequativ) [reply modules/smartadserverBidAdapter.js:145]: Good point, @patmmccann, I've updated the implementation. Thx.

Comment (patmmccann) [reply modules/smartadserverBidAdapter.js:145]: If the publisher specifies midroll starting at 3 minutes (180), you are calling it preroll still?

Comment (krzysztofequativ) [reply modules/smartadserverBidAdapter.js:144]: You are completely right. It's fixed.

Comment (krzysztofequativ) [reply modules/smartadserverBidAdapter.js:144]: Hi @patmmccann, could you review it, please? Thx.

**Review by patmmccann — APPROVED** [review]

**Pull request merged**